<launch>
  <node pkg="epos" exec="epos">
    <param from="$(find-pkg-share osp_bringup)/config/osp_bringup.yaml" />
  </node>

  <node name="dynamixel" pkg="dynamixel_h54" exec="dynamixel_h54">
    <param from="$(find-pkg-share osp_bringup)/config/osp_bringup.yaml" />
  </node>

  <node pkg="osp_bringup" exec="osp">
    <remap from="/command_crawler_left" to="/epos/motor1/move_with_velocity" />
    <remap from="/command_crawler_right" to="/epos/motor2/move_with_velocity" />

    <remap from="/command_flipper_left_front" to="/dynamixel/motor4/command_angle" />
    <remap from="/command_flipper_right_front" to="/dynamixel/motor2/command_angle" />
    <remap from="/command_flipper_left_back" to="/dynamixel/motor3/command_angle" />
    <remap from="/command_flipper_right_back" to="/dynamixel/motor5/command_angle" />
  </node>
</launch>